client/components/domains/register-domain-step/index.jsx

```diff
@@ -541,10 +539,20 @@ const RegisterDomainStep = React.createClass( {
 				severity = 'info';
 				break;
 			case 'not_registrable':
-				if ( domain.indexOf( '.' ) ) {
-					message = this.translate( 'Sorry but %(domain)s cannot be registered on WordPress.com.', {
-						args: { domain: domain }
-					} );
+				const tldIndex = domain.lastIndexOf( '.' );
```

Comment:
Yes, I realized that it will do that for SLDs - but the sentence remains true. The domains "ends with .uk". I'm not familiar with any SLD which we don't allow registrations but allow registrations for the TLD of that SLD (ugh, hope that's clear). So, for `co.uk` we don't support `.co.uk` nor do we support `.uk`.
Why not just use the SLD in the notice? I can't figure out an easy way to reliably extract SLDs, if present. Picking just the TLD is safer.